mobile/src/common/types/navigation/navigation-item.type.ts

```diff
@@ -7,6 +8,7 @@ type NavigationItem = {
   permissions: PermissionKey[];
   component: React.FC;
   isAuthRequired: boolean;
+  options?: NativeStackNavigationOptions;
```

Comment:
options should not be a part of common NavigationItem type, cause we can have different options for different types of navigation, but this type we use for all navigation types. To add this property you need to create a separate type that will extend NavigationItem, like this is done for DrawerNavigationItem
```suggestion
```